src/components/homePage.js

```diff
@@ -54,34 +54,36 @@ const HomePage = () => {
           onChange={handleSearch}
         />
       </div>
-      {filteredCharacters && filteredCharacters.map((character) => (
-        <div key={character.id} className="all">
-          <div className="pic">
-            <a href={`/details/${character.id}`}>
-              <img className="picture" src={character.image} alt="portrait" />
-              <div className="card">
-                <h2 className="card-name">{character.name}</h2>
-                <p className="card-p">{character.planet}</p>
-                <p className="text">
-                  {character.status === 'Alive' && (
+      <div className="home">
+        {filteredCharacters && filteredCharacters.map((character) => (
+          <div key={character.id} className="all">
+            <div className="pic">
+              <a href={`/details/${character.id}`}>
+                <img className="picture" src={character.image} alt="portrait" />
+                <div className="card">
+                  <h2 className="card-name">{character.name}</h2>
+                  <p className="card-p">{character.planet}</p>
+                  <p className="text">
+                    {character.status === 'Alive' && (
                     <FaCircle className="text-green" />
-                  )}
-                  {character.status === 'unknown' && (
+                    )}
+                    {character.status === 'unknown' && (
                     <FaCircle className="text-zinc" />
-                  )}
-                  {character.status === 'Dead' && (
+                    )}
+                    {character.status === 'Dead' && (
                     <FaCircle className="text-red" />
-                  )}
-                  {character.status}
```

Comment:
[REQUIRED] Good job dividing the cards into two sections but in order for your design to look more like the template we you need to remove the spaces between and at the side of the cards.

![image](https://github.com/vanina123/react-capstone/assets/62938735/72603e1a-a29b-4924-8cba-806455111f0f)